### ISSUE TITLE
NaN loss during training

### ISSUE THREAD

**Issue by pikruse** [zhanxw/MB-GAN#7]
Hi,
I am trying to run this model in order to simulate data for a project I am working on. Unfortunately, when I attempt to run train_demo.py, the model loss is NaN for all losses; it begins as NaN and never changes. The only edits I have made to the model are replacing the inheritance of the `_Merge` layer with the `Add` layer and disabling eager execution. However, I am unable to replicate even the published test results. Thanks!

**Comment by zhanxw** [comment]
Can you first check the versions:

Programming language: Python (version 3.6.8), R (version 3.6.0)
Other requirements: Tensorflow (version 1.14.0), Keras (version 2.2.4)

Then you may need to debug to see which line first produced NaN. 
On our computation environment, we did not observe NaN.
